src/main/proto/Report.proto

```diff
@@ -36,7 +36,7 @@ message ReportLineItem {
 }
 
 message AssignReport {
-    string channel = 1;
+    repeated string channel = 1;
```

Comment:
`channels` should be a more suitable variable name for list

Reply:
this will be renamed later in all the project notification and flair-bi...since our code base is quite behind...so please approve it for now...will ask @khushbu-maurya to rename in notification center apis and tables

Reply:
this is a breaking change so flair bi won't compile after this change. Do you have a flair bi pull request ready that will address this change?

Reply:
also not sure why not rename the variable as well since you are touching it anyway. It seems like we will accumulate a tech debt for no reason.

Reply:
this change is required in notification as well

Reply:
yes i have PR ready

Reply:
> also not sure why not rename the variable as well since you are touching it anyway. It seems like we will accumulate a tech debt for no reason.

I would still suggest to do it, do not really see a reason of not doing it since you are changing the field type anyway

Reply:
https://github.com/viz-centric/flair-notifications/pull/94#pullrequestreview-337315793

Reply:
Above PR & this

https://github.com/viz-centric/flair-bi/pull/366/

Reply:
@admin-vizcentric...leaving this to you now...would u like to get this delayed for one day to rename this field in all projects